flyway/postgre/migrations/sql/V1_3__Create_lesson2_tables.sql

```diff
@@ -0,0 +1,68 @@
+CREATE TYPE type_status AS ENUM ('active', 'deactivated', 'deleted');
+CREATE TYPE type_sex AS ENUM ('male', 'female');
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE TABLE IF NOT EXISTS sso_user (
+    user_id SERIAL PRIMARY KEY,
```

Comment:
Cần thêm generate tự tăng
```suggestion
    user_id SERIAL PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```